File: day2/day-2.py

def formatPrice(price):
	

	if not isinstance(price, (int, float)) or price < 0:
		return 'Invalid price'

	return f'${price:,.2f} '


print(formatPrice(25000))
print(formatPrice(49.5))
print(formatPrice(0))
print(formatPrice(-20))
print(formatPrice('abc'))

File: day2/day-2.php
<?php

function formatPrice($price)
{

	if(!is_numeric($price) || $price < 0){

		return 'Invalid price';
	
	}

	$price = number_format((float) $price, 2, '.', ',');

	return '$' . $price;

}

echo formatPrice(45000);

?>